GitHub/GitHub_Get_Closed_PR_by_repo.ipynb

```diff
@@ -0,0 +1,264 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "62b53981-542a-4c3d-b7a1-053791ba7e2d",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "<img width=\"10%\" alt=\"Naas\" src=\"https://landen.imgix.net/jtci2pxwjczr/assets/5ice39g4.png?w=160\"/>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5c199c98-a244-429c-8fd3-41ae0e06efc7",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "# GitHub - Get Closed PR by repo"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cd9544a0-4ef4-4250-a4cc-3bded5ccbb18",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "**Tags:** #github #pygithub #closedpr #repo #api #python"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a001fffd-d496-4194-80a4-5872acbd9452",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "**Author:** [Florent Ravenel](https://www.linkedin.com/in/florent-ravenel/)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d2f016ec-10a6-46e2-9730-4e111c10371a",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "**Description:** This notebook will show how to get closed PR by repo using pygithub library."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9cb89356-d105-4061-90b0-5653078e590f",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "**References:**\n",
+    "- [PyGithub Documentation](https://pygithub.readthedocs.io/en/latest/)\n",
+    "- [GitHub API Documentation](https://developer.github.com/v3/)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3833753b-30f1-4254-9cf3-b498fde5de10",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "## Input"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "50f9d1e9-6430-4e80-836f-2f905af104c9",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "### Import libraries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "21788077-c209-4ab5-b047-7ce1b2f13e0f",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import json\n",
+    "from github import Github\n",
```

Comment:
<img width="1230" alt="Screenshot 2023-02-24 at 6 42 20 PM" src="https://user-images.githubusercontent.com/81156510/221187080-1d81a59a-4e98-4f3e-855e-a4593ecddc77.png">

Getting Import error. Also this doesn't work in Python 3.7+ as per this [comment](https://github.com/PyGithub/PyGithub/issues/856#issuecomment-453809986).

If you are fine then let's just get the PR merged.

Reply:
@srini047, I just tested it again and it works on my Naas. Let me check on your naas account why it is not working.

Reply:
@FlorentLvr Any updates on this?